definitions/features/pulpcore.rb

```diff
@@ -20,6 +20,19 @@ def services
     ]
   end
 
+  def pulpcore_migration_services
+    [
+      system_service('rh-redis5-redis', 5),
+      system_service('pulpcore-api', 10),
+      system_service('pulpcore-content', 10),
+      system_service('pulpcore-resource-manager', 10),
```

Comment:
How about breaking these out in a private method since it's the same as above and combined as needed in `services` and `pulpcore_migration_services`?